### ISSUE TITLE
Usability recommendations

### ISSUE THREAD

**Issue by Jakobovski** [croian/mouseless-issues#34]
### Problem Description

1. Replace `AltRight` with the name for the mac key. On mac its the `OptionRight` key not the `AltRight` key. 
2. Clicking with the mouse should close the overlay
3. By default all the keyboard keys Q->P, A->L, Z->. should be in the sub-grid. If its too much text then dont display them all but interpolate between them. So if i press N and N is not in the subgrid make it halfway between M and B on the screen

### Proposed Solution

See above

**Comment by vlnn** [comment]
I agree with 1 and 2, but 3 by default will break some of my current setups that are not involving layouts with `n` between `b` and `m` — it sounds like perfect optional feature though!

**Comment by croian** [comment]
nice ideas.  will do 1. i can see why 2 would be desirable, and don't really see a downside, so will probably do it as well

is the goal of 3 to have more precision in the subgrid?  i think this might come down to preference -- my thinking with the current default is that its subcells are probably precise enough for the vast majority of clicks, but not having to reach to the middle columns with the index finger (particularly T and Y) is kind of nice.  either way, when i implement presets for different config fields/sections, a 30 key subgrid will definitely be one of them 👍

**Comment by croian** [comment]
(1) is implemented in v0.2.2 (in the default config, the names in your saved configs will persist until changed manually), going to rename / narrow the scope of this one down to (2)
